content/en/getting_started/integrations/_index.md

```diff
@@ -93,37 +93,52 @@ To validate your Agent and integrations configuration, [run the Agent's `status`
 
 Installing more than one integration is a matter of adding the configuration information to a new `conf.yaml` file in the corresponding `<INTEGRATIONS>.d` folder. Look up the required parameters for the new integration from the `conf.yaml.example` file, add it into the new `conf.yaml` file, and then follow the same steps to validate your configuration.
 
+## Autodetected integrations
+
+If you set up [process collection][27], Datadog will autodetect technologies running on your hosts so you can easily identify Datadog integrations that can help you monitor these technologies. These auto-detected integrations are displayed in a section of the Integrations page:
+
+{{< img src="getting_started/integrations/autodetected-integrations2.png" alt="Autodetected integrations"  style="width:80%;">}}
+
+Each integration has one of two status types:
+
+- **+ Detected**: This integration is not enabled on any host(s) running it.
+- **✓ Partial Visibility**: This integration is enabled on some, but not all hosts are running it.
+
+Hosts that are running the integration, but where the integration is not enabled, can be found in the Hosts tab of the integration modal:
```

Comment:
is modal a word customers will be familiar with? I don't believe we use it elsewhere.